### DISCUSSION TITLE
[v1][windows] Why is the taskbar icon gone even after the window is hidden?

### DISCUSSION THREAD

**Discussion by ayangweb** [tauri-apps/tauri#11107] [Q&A] [answered]
Why is the taskbar icon gone even after the window is hidden? 

I'd like to have the taskbar icon resident and click on it to show the window directly!

https://github.com/user-attachments/assets/85875ee2-f979-47e1-8903-5e0994985209

**Comment by amrbashir** [comment, marked as answer]
That's how hiding a window works in the OS, alternatives would:
- minimize the window instead (keeps the indicator that the window is still open)
- hide the window but pin your app icon to the taskbar, in which case clicking it, will launch a new instance of your app then you can use single-instance plugin to prevent the new instance from launching and just show the window again from the old instance.

**Comment by ayangweb** [comment]
Okay, thanks, I got it!
